scripts/run_mlp.py

```diff
@@ -12,29 +12,29 @@
     f.close()
        
 if cfg.DISPLAY_CS:
-    raw_input("Press Enter to display the contact sequence ...")
+    input("Press Enter to display the contact sequence ...")
```

Comment:
If you want this to still work in python 2 (once again, we can also decide that python 2 is no more supported), we have to add:
```python
try:
    # Python 2
    input = raw_input
except NameError:
    pass
```

Reply:
done.